web/src/OrganizationListPage.js

```diff
@@ -273,7 +273,9 @@ class OrganizationListPage extends BaseListPage {
       value = params.passwordType;
     }
     this.setState({loading: true});
-    OrganizationBackend.getOrganizations("admin", params.pagination.current, params.pagination.pageSize, field, value, sortField, sortOrder)
+    (Setting.isAdminUser(this.props.account) ?
+      OrganizationBackend.getOrganizations("admin", params.pagination.current, params.pagination.pageSize, field, value, sortField, sortOrder) :
+      OrganizationBackend.getOrganization("admin", this.props.account.owner, true))
```

Comment:
should not call `getOrganization` here, filter the result in backend for `getOrganizations()`